syntax = "proto3";
package countries;
service Country {
  rpc Search (CountryRequest) returns (CountryResponse) {}
}
message CountryRequest {
  string name = 1;
}
message Currencies {
  string code = 1;
  string name = 2;
  string symbol = 3;
}
message CountryResponse {
  string name = 1;
  string alpha2Code = 2;
  string capital = 3;
  string subregion = 4;
  int32  population = 5;
  string nativeName = 6;
  repeated Currencies currencies = 7;
}